src/components/Questionnaire/QuestionTypes/MedicationStatementQuestion.tsx

```
@@ -81,12 +90,47 @@ const MEDICATION_STATEMENT_INITIAL_VALUE: MedicationStatementRequest = {
+  },
+} as const;
+
+export function validateMedicationStatementQuestion(
+  values: MedicationStatementRequest[],
+  questionId: string,
+): QuestionValidationError[] {
+  return values.reduce((errors: QuestionValidationError[], value, index) => {
+    // Skip validation for medications marked as entered_in_error
+    if (value.status === "entered_in_error") return errors;
+    return [
+      ...errors,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)